agent/serviceinfobroker/service_info_broker.go

```diff
@@ -222,9 +222,30 @@ func (sib *ServiceInfoBroker) getPostgreSQLInfo(ctx context.Context, dsn string,
 	var version string
 	if err = db.QueryRowContext(ctx, "SHOW /* agent='serviceinfobroker' */ SERVER_VERSION").Scan(&version); err != nil {
 		res.Error = err.Error()
+		return &res
```

Comment:
It's not a miss, and there was no intention to return the result here :)

Reply:
Yes, but I am running another query after this one. That is why I returned it here.

Reply:
The absence of version is not critical, that's why we can continue to the next query.

Reply:
But what if another query will end up with error? Error type is string. So join them? Wrap two not related error?

Reply:
Only idea I got is move all queries above and let version query as last one.